### PR TITLE
More typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`

### CODE REVIEW

**Comment by luzpaz** [comment]
@caryoscelus would you care to review ?

**Comment by caryoscelus** [comment]
@luzpaz looks good to me

**Review by bryceharrington — APPROVED** [review]

**Comment by luzpaz** [comment]
@tweenk care to review ?

File: src/2geom/concepts.h [hunk 1]
```diff
@@ -139,7 +139,7 @@ template <typename T>
 struct EqualityComparableConcept {
     T a, b;
     bool bool_;
-    void constaints() {
+    void constraints() {
```
Comment (luzpaz) [inline src/2geom/concepts.h:142]: please  review

**Pull request merged**